app/plugin_api_test.go

```diff
@@ -72,8 +72,10 @@ func setupMultiPluginApiTest(t *testing.T, pluginCodes []string, pluginManifests
 	require.NoError(t, err)
 	webappPluginDir, err := ioutil.TempDir("", "")
 	require.NoError(t, err)
-	defer os.RemoveAll(pluginDir)
-	defer os.RemoveAll(webappPluginDir)
+	t.Cleanup(func() {
+		os.RemoveAll(pluginDir)
+		os.RemoveAll(webappPluginDir)
+	})
```

Comment:
Maybe this is causing the ci failure

Reply:
Indeed it was, but for entirely unexpected reasons -- the code used to delete the `pluginDir` right away. This was preventing the usual app plugin synchronization code from running, which in turn would try to disable plugins not explicitly enabled.

I've fixed this by updating the app configuration to explicitly enable all of these plugins. Let's see how the CI fares.